.github/workflows/node-create-package-file/action.yml

```diff
@@ -13,13 +13,15 @@ inputs:
             - amazon-linux
             - macos-latest
             - ubuntu-latest
+            - ubuntu-latest-musl
     named_os:
         description: "The name of the current operating system"
         required: false
         default: "linux"
         type: string
         options:
             - linux
+            - linux-musl
```

Comment:
Please see the comments I wrote on the other PR - we shouldn't use `linux-musl` here